README.md

```diff
@@ -70,8 +70,8 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- coming soon after aprroved ....
```

Comment:
- [ ] You should deploy the application using Render, Netlify, or GitHub Pages.
- [ ] The project is deployed properly, the home and detail page works as expected.
- [ ] A link to the deployed project is present in the README.